dist/polyfill.js

```diff
@@ -277,6 +286,33 @@ Promise.all = function(arr) {
   });
 };
 
+Promise.any = function(arr) {
```

Comment:
Let's add this code to both index.js and polyfill.js. Might make sense to abstract this function into a separate file

Reply:
Sure, let me try, I'll update this request when it will be done 👌🏻